src/main/java/seedu/address/model/DateTime.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.DateTimeException;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
```

Comment:
I was actually trying to implement DateTime as well. 

I think using java.time.LocalDateTime is better since java.util.Date is deprecated. Although I see that you didn't actually use Date at all.

Reply:
Yeah was an incorrect import, will edit